packages/common/config/apis.ts

```diff
@@ -180,7 +180,7 @@ export const apiConfigurations: ReadonlyArray<Readonly<APIConfiguration>> = [
   {
     id: "automation-hub",
     displayName: "Automation Hub",
-    description: "Fetch, Upload, Organize, and Distribute Ansible Collections",
+    description: "Upload, Organize, and Distribute Ansible Collections",
```

Comment:
whoops. Missed the `Fetch, `

```suggestion
    description: "Fetch, Upload, Organize, and Distribute Ansible Collections",
```